packages/account/src/Sections/Assessment/FinancialAssessment/__tests__/financial-assessment.spec.tsx

```diff
@@ -72,4 +72,32 @@ describe('<FinancialAssessment/>', () => {
             expect(screen.getByText('Anticipated account turnover')).toBeInTheDocument();
         });
     });
+
+    it('should render FinancialAssessment component without occupation field when Employment status is self employed', async () => {
+        WS.authorized.storage.getFinancialAssessment = jest.fn(() =>
```

Comment:
just a doubt, is reassigning the right way? or are there any functions like mockFunctionValue?

Reply:
I did try that. But it didn't work. Since it was urgent I went with this approach. Please suggest if u get a solution